### DISCUSSION TITLE
Dialog for changing setting only pops up a message with current state

### DISCUSSION THREAD

**Discussion by cpapplefamily** [wpilibsuite/2024Beta#36] [General Technical]
I often turn off the Start RioLog on Deploy Setting. When I select the {W} icon and select the [WPILib: Change Auto Start RioLog on Deploy Setting] I get a popup the tells me it is currently enabled. 

The old behavior had multiple options for enable/disable for project or globally. 

Yes I can got to File>Preferences>Settings>Extensions>WPILib configuration>... to change but then why even have a nonfunctional option in the other dialog box.

**Comment by sciencewhiz** [comment]
This appears to be a regression in VS Code. Running the 2024 extension in VS Code 1.72.2 (from 2023), the dialog works, and there's been no changes in the extension in that area in years.

**Comment by sciencewhiz** [comment]
Opened https://github.com/wpilibsuite/vscode-wpilib/issues/637

**Reply by sciencewhiz** [reply]
Fixed in https://github.com/wpilibsuite/vscode-wpilib/pull/640
